docs/okeps/okep-5088-evpn.md

```diff
@@ -0,0 +1,893 @@
+│     +-----------------+                                                                                 │         │  
+│     |   vxlan0        |  local 100.64.0.1 dstport 4789 external                                         │         │  
+│     +-----------------+                                                                                 │         │  
+│            │  (VTEP)                                                                                    │         │  
+│            └────────────────────────────────────────────────────────────────────────────────────────────┘         │  
+│                                        Encapsulated EVPN traffic (VNI 100 & 101)                                  │  
+│                                             via 100.64.0.1 <-> 100.64.0.x peers                                   │  
+└────────────────────────────────────────────────────────────────────────────────────────────────────────────────── │
+```
+
+The MAC address for the bridge is unique and will be automatically generated by OVN-Kubernetes. This MAC address is
+known as the "router-mac" and is used by Type 5 routes to know what the destination MAC of the next hop should be. The
+MAC can be the same on every node, but must be different per UDN.
+
+Furthermore, bridge and VXLAN link names may also change and will be decided by OVN-Kubernetes. 
+
+While the IP-VRF uses pure routing to transmit traffic over the EVPN fabric, MAC-VRF relies on layer 2.
+For that reason, the layer 2 OVN network needs to be extended into the EVPN fabric.
+To do that, we connect br-int to the linux bridge for the MAC-VRF. This will allow layer 2 traffic to
+travel through br-blue and then eventually into the EVPN fabric via the VNID 100. This enables a user to disable the
+Geneve overlay and allow L2 communication between UDNs on different nodes via the MAC-VRF:
+
+<a id="l2evpn-anchor"></a>
+![](images/l2evpn.png)
+
+Note, the Layer 2 domain for the MAC-VRF may be extended into the provider's physical network, and not just extended across
+Kubernetes nodes. This allows for VM migration and other layer 2 connectivity between entities external to the cluster
+and entities within.
+
+ovnkube-controller will be responsible for configuring OVN, including the extra OVS internal port attached to the worker logical
+switch.
+
+In addition to VTEP IP allocation, ovnkube-cluster-manager will be responsible for generating FRR-K8S config to enable
+FRR with EVPN. The config for the above example would look something like this:
+
+```bash
+vrf blue
+ vni 101
+ rd 65000:101
+exit-vrf
+!
+router bgp 65000
+ !
+ ! Peer with spine using eBGP
+ neighbor 192.168.122.12 remote-as 65001
+ !
+ address-family ipv4 unicast
+  network 100.64.0.1/32
+ exit-address-family
+ ! MAC-VRF config start
+ address-family l2vpn evpn
+  evpn vni 100 l2
+    rd 65000:100
+    route-target import 65000:100
+    route-target export 65000:100
+    bridge-domain 100
+  ! 
```

Comment:
nit on syntax
```
address-family l2vpn evpn
  vni 100
    rd 65000:100
    route-target import 65000:100
    route-target export 65000:100
```

I don't see `bridge-domain 100` as valid syntax, I haven't needed it for any of my labs so I think we can do without?